### PR TITLE
Add method_exists to avoid fatal error

### PR DESCRIPTION
**Pull request by ruudvanderweijde**
Related error:
```
  [Symfony\Component\Debug\Exception\UndefinedMethodException]
  Attempted to call an undefined method named "getCustomTemplate" of class "Doctrine\DBAL\Migrations\Configuration\Configuration".
```

This commit can be reverted when https://github.com/doctrine/migrations v1.6 is released.

Relates to https://github.com/doctrine/DoctrineMigrationsBundle/issues/207
Relates to https://github.com/doctrine/DoctrineMigrationsBundle/compare/v1.2.1...v1.3.0#commitcomment-24024900

### CODE REVIEW

**Comment by lcobucci** [comment]
I'll configure travis in a bit, thanks for your help

**Comment by lcobucci** [comment]
@ruudvanderweijde config added on #214 please rebase your branch to sync things

**Comment by ruudvanderweijde** [comment]
@lcobucci I've rebased and added a test, can you check again?
Did you already set up a travis integration?

File: Tests/Command/DoctrineCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Doctrine\Bundle\MigrationsBundle\Tests\DependencyInjection;
+
+use Doctrine\Bundle\MigrationsBundle\Command\DoctrineCommand;
+use Doctrine\DBAL\Migrations\Configuration\Configuration;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+
+class DoctrineCommandTest extends \PHPUnit\Framework\TestCase
```
Comment (j0k3r) [inline Tests/Command/DoctrineCommandTest.php]: `\PHPUnit_Framework_TestCase` might be better

**Comment by bendavies** [comment]
surely the composer deps also need to be updated?
`"doctrine/migrations": "^1.6"`

File: Tests/Command/DoctrineCommandTest.php (hunk 1 above)
Comment (Jean85) [reply Tests/Command/DoctrineCommandTest.php]: Maybe it's better to leave it as it is and require PHPUnit 4.8.35+.

Comment (Soullivaneuh) [reply Tests/Command/DoctrineCommandTest.php]: Maybe it's even better to import the class namespace. :wink:

**Comment by ruudvanderweijde** [comment]
Thanks @lcobucci, I was just about to do it :)

**Comment by lcobucci** [comment]
@ruudvanderweijde 🚢  thanks for your contribution!

**Pull request merged**